Tools/gittools/pre-commit.py

```diff
@@ -17,17 +17,14 @@
 
 class AP_PreCommit(object):
 
-    def __init__(self):
-        pass
-
     @staticmethod
     def progress(message):
-        print("***** %s" % (message, ))
+        print(f"***** {message}")
```

Comment:
While we are Python3-only, I was actually shying away from f-strings straight-away.  Oh well.